api/serializers.py

```diff
@@ -39,43 +39,61 @@
 )
 from notifications.models import Subscription
 
+
 class DisasterTypeSerializer(serializers.ModelSerializer):
     class Meta:
         model = DisasterType
         fields = ('name', 'summary', 'id',)
 
+
 class RegionSerializer(serializers.ModelSerializer):
     class Meta:
         model = Region
         fields = ('name', 'id', 'region_name')
 
 
+class CountryCsvSerializer(serializers.ModelSerializer):
```

Comment:
`CsvSerializer` sounds like it provides CSV. Maybe use `CountryFlatSerializer` or `CountryTableauSerializer`?
`CountryTableauSerializer` should be used, as this structure shouldn't be used/changed for go-frontend and provides scope into Tableau.
@batpad @GregoryHorvath What do you guys think?

Reply:
Thanks for the suggestion @thenav56 yea I was thinking about this too. It originated from the idea that I'd just use the `*CsvSerializer` for most endpoints and that would be good enough and it worked well for https://github.com/IFRCGo/go-api/pull/762. Only that most endpoints didn't have a different serializer for CSVs so where I needed it I just made one.

If you know a way to better extend out from the original serializers rather than copy-pasting them, changing 1 line and calling them `*CsvSerializer` please share :slightly_smiling_face: .
Something that comes into my mind is like (pseudo code):
```python
@overrides XYSerializer
class XYTableauSerializer():
    event = EventTableauSerializer()
    # and the rest is automatically there from the XYSerializer of course
```

cc @batpad

Reply:
There might be issues if we override. For instance, if a non-flat field is added to the parent serializer then it will be also added to `*TableauSerializer`. Best will be the way you are doing right now. The reason I recommend using `*TableauSerializer` is that we will have the freedom to change and sync it with the web connector and it will not affect users who are using the CSV/Flat API directly.

Reply:
https://github.com/IFRCGo/go-api/pull/830/commits/74ea79738d0e3c6f306f1b4202915e4b3bb3c806